### ISSUE TITLE
Add informative note to tts:fontStyle recommending use of tts:shear instead of tts:fontStyle="italic" for Japanese text

### ISSUE THREAD

**Issue by jmccrossan-fox** [w3c/ttml2#1120]
Per [CSS](https://developer.mozilla.org/en-US/docs/Web/CSS/font-style)
-  "italic" selects a particular version of the font, which might not exist. 
- when an "italic" version of the font does not exist, CSS silently converts "italic" to "oblique", which, by default, maps to a recommended shear of 14 deg.

Many Japanese fonts do not have an "italic" version. 

Consider adding an informative note in the definition of [tts:fontStyle](https://w3c.github.io/ttml2/index.html#style-attribute-fontStyle) highlighting the above and recommending that, to avoid indeterministic results, the content author specify [tts:shear](https://w3c.github.io/ttml2/index.html#style-attribute-shear) on Japanese text instead of tts:fontStyle="italic".

**Comment by nigelmegitt** [comment]
@jmccrossan-fox Thank you for raising this issue!

**Comment by skynavga** [comment]
To date, we have not made stylistic recommendations in TTML2; nor have we singled out treatment of content from specific languages. Rather, we have defined features that are useful across a range of languages usage. You fail to note that there also exists an `oblique` value for `tts:fontStyle` which would be a closer fit. Finally, suggesting the use of just `tts:shear` is problematic, since it doesn't apply to `span`, which is what `tts:fontStyle` applies to. So overall, I am opposed to this suggestion, both in general and in its particulars.

**Comment by skynavga** [comment]
I should add that it is the responsibility of the content author to select fonts that correspond to the styles they employ, and that, if they want to specify an "italic" form of a CJK font, then they are free to do so. They can always embed the font directly or reference it as an external resource using the `font` element.

**Comment by palemieux** [comment]
@cconcolato Any input on italic vs oblique vs shear?

**Comment by skynavga** [comment]
I would not object to moving this issue to IMSC, which does have precedent for making author recommendations. Also, as a point of note, CAP2TT [1] maps inline shear to oblique style when generating imsc11 output from Lambda CAP (a popular Japanese subtitle format) since imsc11 does not support `tts:fontShear`.

[1] https://github.com/skynav/ttt/issues/239

**Comment by palemieux** [comment]
> Also, as a point of note, CAP2TT [1] maps inline shear to oblique style when generating imsc11 output from Lambda CAP (a popular Japanese subtitle format) since imsc11 does not support tts:fontShear.

Presumably `tts:shear` could be used if "oblique" is applied to all text within a paragraph, including ruby text.

**Comment by skynavga** [comment]
@palemieux yes, in Lambda CAP, one can apply 斜め (i.e., obliquing) with a specified angle to either (1) a "screen" (= paragraph in TTML) or (2) inline text (= span in TTML); CAP2TT translates the former to `tts:shear` and the latter to `tts:style='oblique'` when generating imsc11 output; for ttml2 output, it translates both to `tts:fontShear`;

**Comment by cconcolato** [comment]
I'm fine with adding the note to IMSC.

**Comment by cconcolato** [comment]
@palemieux would you be ok moving this issue to IMSC?

**Comment by skynavga** [comment]
Do we have consensus to move it? If so, then one of the editors can do the mechanics of the move.

**Comment by nigelmegitt** [comment]
@skynavga I think we have consensus, but I haven't tested that yet. Will add to agenda for this week, and we can action it after then if need be.
